src/authz-smart/smartHandler.ts

```diff
@@ -8,31 +8,135 @@ import {
     AuthorizationRequest,
     AuthorizationBundleRequest,
     AllowedResourceTypesForOperationRequest,
+    SystemOperation,
+    TypeOperation,
 } from 'fhir-works-on-aws-interface';
-import { SMARTConfig } from './smartConfig';
+import axios from 'axios';
+import { LaunchType, ScopeType, SMARTConfig } from './smartConfig';
 
 // eslint-disable-next-line import/prefer-default-export
 export class SMARTHandler implements Authorization {
+    static readonly CLINICAL_SCOPE_REGEX = /^(patient|user|system)\/(\w+|\*)\.(read|write|\*)$/;

```

Comment:
`w+` may be too open. I believe ResourceTypes are letters only
`patient/Observation_77.read` shouldn't match